charts/sscs-bulk-scan/values.yaml

```
@@ -19,3 +17,16 @@ java:
     ROOT_LOGGING_LEVEL: INFO
     LOG_OUTPUT: single
     IDAM_OAUTH2_REDIRECT_URL: "https://evidence-sharing-preprod.sscs.reform.hmcts.net"
+    IDAM_URL: "https://idam-api.aat.platform.hmcts.net"
+    IDAM_OAUTH2_CLIENT_ID: "sscs"
+    IDAM_S2S_AUTH_MICROSERVICE: "sscs_bulkscan"
+    LOGBACK_REQUIRE_ALERT_LEVEL: "false"
+    LOGBACK_REQUIRE_ERROR_CODE: "false"
+    WEBSITE_LOCAL_CACHE_OPTION: "Always"
+    WEBSITE_LOCAL_CACHE_SIZEINMB: "300"
+    WEBSITE_NODE_DEFAULT_VERSION: "8.11.1"
+    WEBSITE_SLOT_POLL_WORKER_FOR_CHANGE_NOTIFICATION: "0"
+    NODE_PATH: "D:\\home\\site\\wwwroot"
+    IDAM_S2S_AUTH: "http://rpe-service-auth-provider-aat.service.core-compute-aat.internal"
```

https://github.com/hmcts/draft-store/blob/master/charts/draft-store-service/values.yaml#L12
```suggestion
    IDAM_S2S_AUTH: http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
```